cpp/test/IceStorm/stress/Subscriber.cpp

```diff
@@ -235,7 +235,14 @@ class Subscriber final : public Test::TestHelper
 void
 Subscriber::run(int argc, char** argv)
 {
-    Ice::CommunicatorHolder ich = initialize(argc, argv, make_shared<Ice::Properties>(vector<string>{"IceStormAdmin"}));
+    InitializationData initData;
+    initData.properties = make_shared<Ice::Properties>(vector<string>{"IceStormAdmin"});
+    initData.properties = createProperties(argc, argv, initData.properties);
```

Comment:
I could not figure out a simpler way to do this. The API isn't great.

Reply:
Maybe another `createTestProperties` that takes a defaults (Properties)

Reply:
So far we only need it once for this test.